python/python/tests/test_embeddings.py

```diff
@@ -198,7 +198,6 @@ class TestSchema(LanceModel):
         {
             "text": ["hello world", "goodbye world"],
             "val": [1, 2],
-            "not-used": ["s1", "s3"],
```

Comment:
@eddyxu You added this test asserting we could insert junk columns and they would be ignored. (#1818) Is that important behavior to keep? This PR breaks it.